voter/tests/test_api.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from unittest.mock import Mock, patch
+
+from django.test import TestCase
+
+from voter import models
+from voter.views import changes
+
+
+class APIChangesTests(TestCase):
+
+    def makeRequest(self, params):
+        request = Mock()
+        request.GET = params
+        return request
+
+    def makeChange(self, ncid, data):
```

Comment:
Nitpick: Could we use underscores instead of camelCase? :) (and for makeRequest, as well)